language/moe/evaluate_mbxp.py

```diff
@@ -0,0 +1,208 @@
+import argparse
```

Comment:
This script is quite problematic. Sometimes for some reasons one/few languages does not work at all. I believe that the script should make some soft of sanity check trying to evaluate one, hardcoded answer for each language and print error explaining why it failed.

Reply:
I think we can merge and open an issue for this script